RELEASE_HISTORY.md

```diff
@@ -0,0 +1,84 @@
+# Zowe CLI Releases
+
+Zowe follows a regular release schedule with major versions released every two years and minor versions approximately every six weeks. Full details regarding the Zowe release schedule are available on [zowe.org](https://www.zowe.org/download#timeline) and the Zowe Community [Github](https://github.com/zowe/community/blob/master/Technical-Steering-Committee/release.md).
+
+## Zowe Release Schedule
+
+### Zowe v3.x LTS Releases
+|  Version | Release Date |  Status    | Release Notes |
+|:--------:|:------------:|:----------:|:-------------:|
+|  v3.0.0  | 2024-09      | **Active** | [Link](#)     |
+
+### Major Release Timeline
+
+![Zowe Major Releases](https://raw.githubusercontent.com/zowe/zowe.github.io/master/assets/img/major_releases.webp)
+
+### Version Timeframes
+
+- **Active**: Each major version remains in this phase for 2 years, receiving regular updates and new features.
+- **Maintenance**: Following the Active phase, each major version remains in this phase for an additional 2.5 years, receiving only critical fixes and security patches.
+- **Under Development**: The pre-Active phase where the next major version is prepared. This phase varies in length and is not available for general consumption.
```

Comment:
- replace "where" with "when"
- I would edit the second sentence a little: " ... varies in length and **the version** is not available for ... "